### PR TITLE
Adding spec storage tests to FS

### PR DESCRIPTION
**Pull request by AmarnatReddy**
# Description
Adding spec storage tests to FS
We have made changes to the installation part of specstorage. Earlier we use to mount and run it. 
Now we are downloading it to the client and running the IOs

Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-J7SAWU  -> VDA workload
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-UNZLN9 --> SWBUILD

Complete Log :  http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-7NY5GT

FIO Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-DP9HKX

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by MohitBis — APPROVED** [review]
LGTM

File: tests/cephfs/cephfs_spec_storeage.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
+import os
```
Comment (neha-gangadhar) [inline tests/cephfs/cephfs_spec_storeage.py:1]: can you please fix minor typo in the file name to - "cephfs_spec_storage.py"

File: suites/reef/cephfs/tier-2_cephfs_io.yaml [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+          - ceph-common
+        node: node7
+      desc: "Configure the Cephfs client system 1"
+      destroy-cluster: false
+      module: test_client.py
+      name: "configure client"
+  - test:
+      abort-on-fail: false
+      desc: "Run SWBUILD Workload"
+      module: cephfs_spec_storeage.py
```
Comment (neha-gangadhar) [inline suites/reef/cephfs/tier-2_cephfs_io.yaml:116]: please fix typo in module names for all the tests in suite file to "cephfs_spec_storage.py"

**Review by neha-gangadhar — COMMENTED** [review]
lgtm

**Review by pranavprakash20 — APPROVED** [review]
LGTM

**Review by neha-gangadhar — APPROVED** [review]
approved

**Pull request merged**